### DISCUSSION TITLE
How do I update Beets?

### DISCUSSION THREAD

**Discussion by ladywhiskers** [beetbox/beets#4965] [Q&A] [answered]
This may be a really really obvious question, but how do I update beets?  I can see that heaps of updates have happened on github, but when I update via `pip install -U beets` it appears I have the most recent versions of all the files. But I know that I don't have the new UI etc.

I'm assuming this is because the latest "release" is 1.6.0 which is from 2021, so is the best way to update to download and install the files manually? Or will this break something?

I hope that all made sense!

**Comment by Serene-Arc** [comment, marked as answer]
We're working on getting a protocol in place for doing releases, but right now you have to pull from the github. The following command should work for you.

```bash
pip install -U git+https://github.com/beetbox/beets.git
```

**Comment by isle9** [comment]
> We're working on getting a protocol in place for doing releases, but right now you have to pull from the github. The following command should work for you.
> 
> ```shell
> pip install -U git+https://github.com/beetbox/beets.git
> ```

Is there an ETA for this and/or for the 1.6.1 release?

**Reply by Serene-Arc** [reply]
Not yet but we're discussing a workflow for doing it easily and frequentl right now.
